consul/resource_consul_acl_token_policy_attachment_test.go

```diff
@@ -0,0 +1,170 @@
+				}
+			}
+		}
+	}
+	return nil
+}
+
+func testAccCheckTokenExists(n string, tokenID *string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[n]
+		if !ok {
+			return fmt.Errorf("Not Found: %s", n)
+		}
+
+		tokenID := rs.Primary.Attributes["id"]
+		if tokenID == "" {
+			return fmt.Errorf("No token ID is set")
+		}
+
+		client := getClient(testAccProvider.Meta())
+		_, _, err := client.ACL().TokenRead(tokenID, nil)
+		if err != nil {
+			return fmt.Errorf("Unable to retrieve token %q", tokenID)
+		}
+
+		return nil
+	}
+}
+
+func TestAccConsulACLTokenPolicyAttachment_basic(t *testing.T) {
+	var tokenID string
+
+	resource.Test(t, resource.TestCase{
+		Providers: testAccProviders,
+
+		PreCheck:     func() { testAccPreCheck(t) },
+		CheckDestroy: testAccCheckConsulACLTokenPolicyAttachmentDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testResourceACLTokenPolicyAttachmentConfigBasic,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckTokenExists("", &tokenID),
+					resource.TestCheckResourceAttr("consul_acl_token_policy_attachment.test", "token_id", tokenID),
+					resource.TestCheckResourceAttr("consul_acl_token_policy_attachment.test", "policy", "test"),
+				),
+			},
+			{
+				Config: testResourceACLTokenPolicyAttachmentConfigUpdate,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckTokenExists("", &tokenID),
+					resource.TestCheckResourceAttr("consul_acl_token_policy_attachment.test", "token_id", tokenID),
+					resource.TestCheckResourceAttr("consul_acl_token_policy_attachment.test", "policy", "test"),
+				),
+			},
+			{
+				Config:  testResourceACLTokenPolicyAttachmentConfigBasic,
+				Destroy: false,
+			},
+		},
+	})
+}
+
+func TestAccConsulACLTokenPolicyAttachment_import(t *testing.T) {
+	var tokenID string
+
+	checkFn := func(s []*terraform.InstanceState) error {
+		if len(s) != 1 {
+			return fmt.Errorf("bad state: %s", s)
+		}
+		v, ok := s[0].Attributes["token_id"]
+		if !ok || v != tokenID {
+			return fmt.Errorf("bad token_id: %s", s)
+		}
+		v, ok = s[0].Attributes["policy"]
+		if !ok || v != "test" {
+			return fmt.Errorf("bad policy: %s", s)
+		}
+
+		return nil
+	}
+
+	resource.Test(t, resource.TestCase{
+		Providers: testAccProviders,
+		PreCheck:  func() { testAccPreCheck(t) },
+		Steps: []resource.TestStep{
+			{
+				Config: testResourceACLTokenPolicyAttachmentConfigBasic,
+				Check:  testAccCheckTokenExists("", &tokenID),
+			},
+			{
+				ResourceName:     "consul_acl_token_policy_attachment.test",
+				ImportState:      true,
+				ImportStateCheck: checkFn,
+			},
+		},
+	})
+}
+
+const testResourceACLTokenPolicyAttachmentConfigBasic = `
+resource "consul_acl_policy" "test" {
+	name = "test"
+	rules = "node \"\" { policy = \"read\" }"
+	datacenters = [ "dc1" ]
+}
+
+resource "consul_acl_token" "test" {
+	description = "test"
+	local = true
+
+	lifecycle {
+		ignore_ttributes = ["policies"]
```

Comment:
typo?

Reply:
Yes, the file content was wrong. I just updated it, pushing a new revision.